README.md

```diff
@@ -59,6 +59,10 @@ Here are the options you can give Smokescreen:
    --deny-address value                        Add IP[:PORT] to list of blocked IPs.  Repeatable.
    --allow-address value                       Add IP[:PORT] to list of allowed IPs.  Repeatable.
    --egress-acl-file FILE                      Validate egress traffic against FILE
+   --prometheus-endpoint ENDPOINT              Expose Prometheus metrics on the given endpoint.
```

Comment:
does ENDPOINT default to /metrics? I'm a bit confused by some of the wording here.

Reply:
@alex-kalinowski: https://github.com/sanity-io/smokescreen/pull/1#discussion_r983667787 <- I'm happy to add a `--enable-prometheus` flag, and have the `--prometheus-endpoint` default to `/metrics` - it's probably the cleanest way to structure the config.